src/main/java/uk/gov/hmcts/reform/sscs/model/servicehearingvalues/ScreenNavigation.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.sscs.model.servicehearingvalues;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder(toBuilder = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@AllArgsConstructor
+@NoArgsConstructor
+public class ScreenNavigation {
```

Comment:
Do we need this?

Reply:
I don't know - this was in the schema, so I added it to keep things consistent. Would you like me to remove it?

Reply:
I only asked because in our call yesterday I believe Sean said ScreenNavigation won't be used by SSCS

Reply:
yes, I do remember this. suggest leaving it, so it is schema compliant, but do not populate.